css-three-column-layout/styles.css

```diff
@@ -0,0 +1,51 @@
+body {
+  justify-content: center;
+}
+
+.container {
+  background-color: rgb(67, 129, 229);
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  font-family: lato, sans-serif;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  gap: 2rem;
+}
+
+h2 {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  color: white;
+}
+
+h3 {
+  color: white;
+}
+
+p {
+  color: white;
+  font-size: smaller;
+}
```

Comment:
In these smaller exercises this is okay, but as we get into larger exercises and projects I would recommend using classes for different sections. What happens if you don't want all of the text on your page to be white and smaller? What if you don't want the padding around all of your `h2`s?

Because of the specificity and possible different styling, using classes will help you from having to fight yourself.